TypeCobol/Compiler/Diagnostics/TypeCobolChecker.cs

```diff
@@ -936,16 +937,50 @@ public static void CheckStatement(Node node)
 
     public class ProgramChecker
     {
-        public static void OnNode(Program node)
+        public static void OnNode(Program node, TypeCobolOptions typeCobolOptions)
```

Comment:
Can you move this method in CrossChecker ?
There is nothing related to the TypeCobol language here. It was already wrong before this PR.